.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+Replace this paragraph with a description of what this PR is doing. Include
+links to relevant issues. If you're modifying existing behavior, describe the
+existing behavior, how this PR is changing it, and what motivated the change. If
+you're changing visual properties, consider including before/after screenshots
+(and runnable code snippets to reproduce them).
+
+## Checklist
+
+Before you create this PR confirm that it meets all requirements listed below
+by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick
+review process.
+
+- [ ] I read the [Contributor Guide] and followed the process outlined there for
+      submitting PRs.
+- [ ] My PR includes tests for *all* changed/updated/fixed behaviors (See
+      [Test Coverage]).
+- [ ] All existing and new tests are passing.
+- [ ] I updated/added relevant documentation (doc comments with `///`).
+- [ ] The analyzer (`flutter analyze --flutter-repo`) does not report any
+      problems on my PR.
+- [ ] I read and followed the [Flutter Style Guide], especially
+      [Features we expect every widget to implement].
+- [ ] I signed the [CLA].
+- [ ] I am willing to follow-up on review comments in a timely manner.
+
+## Breaking Change
+
+Does your PR require Flutter developers to manually update their apps
```

Comment:
Maybe add some examples. People may not realize what makes changes breaking (adding enum values, add new asserts etc)

Reply:
Good idea. That should probably go in the wiki though and be linked from here to keep this short-ish.